Src/World/Timing.cpp

```diff
@@ -1,70 +1,78 @@
 #include "Timing.h"
 
-#include <Math/Math.h>
-
 
 // The maximum amount of time the accumulator can store.
-constexpr double MAX_ACCUMULATED_SECONDS = 3.0;
+static constexpr double MAX_ACCUMULATED_SECONDS = 3.0;
+using Clock = std::chrono::high_resolution_clock;
```

Comment:
Prefer namespace aliases

Reply:
std::chrono is the namespace, high_resolution_clock is an object, error is thrown when aliasing.
I have changed references to Clock into their distinct paths, as it was only used in relatively few places.